pod2daemon/csidriver/Dockerfile.amd64

```diff
@@ -38,6 +38,7 @@ COPY --from=ubi /lib64/libc.so.6 /lib64/libc.so.6
 COPY --from=ubi /etc/host.conf /etc/host.conf
 COPY --from=ubi /etc/nsswitch.conf /etc/nsswitch.conf
 
-ADD bin/csi-driver-amd64 /usr/local/bin/csi-driver
+ARG BIN_DIR
+ADD ${BIN_DIR}/csi-driver-amd64 /usr/local/bin/csi-driver
```

Comment:
do we need to repeat this step for the flexvol image too?

Reply:
flexvol does not use SSL libs so making FIPS versions of the image is unnecessary